src/utils/findPersonByName.ts

```diff
@@ -0,0 +1,4 @@
+import { Person } from './../types/Person';
+
+export const findPersonByName = (people: Person[], name: string | null) =>
+  people.find(person => person.name === name);
```

Comment:
The function 'findPersonByName' does not return any value. You should add a return statement to return the result of the 'find' method.